devtools/client/framework/target-from-url.js

```diff
@@ -88,6 +91,27 @@ exports.targetFromURL = Task.async(function*(url) {
       }
       throw ex;
     }
+  } else if (type == "window") {
+    // Fetch target for a remote window actor
+    DebuggerServer.allowChromeProcess = true;
+    try {
+      id = parseInt(id, 10);
+      if (isNaN(id)) {
+        throw new Error("targetFromURL, " +
```

Comment:
Changed to one line. For whitespace generally, this is the style used in DevTools code, which is what is being modified here, so I'll leave it as-is. For code in `/positron`, I am happy to fit whatever style you prefer.